lib/cli-engine.js

```diff
@@ -553,6 +554,9 @@ CLIEngine.prototype = {
                     if (warnIgnored) {
                         results.push(createIgnoreResult(filename));
                     }
+                    if (options.verbose) {
+                        log.info("Ignoring: " + filename);
```

Comment:
So apparently, `console.warn` is just an [alias](https://nodejs.org/api/console.html#console_console_warn_data) for `console.error`. Let's just stick with the latter, since we already have a logging [cover](https://github.com/eslint/eslint/blob/master/lib/logging.js#L23) defined for it.